tardis/io/decay.py

```diff
@@ -60,9 +60,8 @@ def to_materials(self):
 
         Returns
         -------
-            : list
-            list of pyne Materialss
-        :return:
+        list
```

Comment:
colon missing?

Reply:
Jaladh advised  that I should remove the colon in an earlier comment.

Reply:
I am going through and I noticed that a lot of them follow this format, so I will double check on this.